Gaijinizer patch/Troops.txt

```diff
@@ -11406,10 +11427,9 @@ Isn't that right, \\N[1]....?#Triketos
 『表』の私たちが、悪魔にしては大人しい性格だから
 少し油断しちゃったのかしら…？
 > CONTEXT: Troops/297/pages/1/99/Dialogue/enemy13
-Fufu......
-I wonder if we in "surface" have been a little
-offensive because they are decent
-characters for the devil ...?#GT
+If wonder if you let down your guard a little
+because their personalities are so meek for
+demons?#Ron
```

Comment:
May want to relook at this one, you've missed the "Fufu..." and there's a few grammatical errors here...